database/migrations/2022_12_14_103127_add_cts_exam_column.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('training_waiting_list', function (Blueprint $table) {
+            $table->string('cts_theory_exam_level')->nullable();
```

Comment:
Nitpicking here, but `cts_theory_exam_level` wasn't immediately obvious to me what it meant. Maybe `cts_theory_exam_code`?

Reply:
I chose level because that was the corresponding column name from CTS so wanted to keep consistent